lib/snip.c

```diff
@@ -662,6 +698,9 @@ grn_snip_close(grn_ctx *ctx, grn_snip *snip)
        cond < cond_end; cond++) {
     grn_snip_cond_close(ctx, cond);
   }
+  if (snip->cond) {
```

Comment:
Is this needed? It seems that we always have non-NULL `snip->cond`.

Reply:
Fixed.